packages/fakta-inntektsmelding/src/InntektsmeldingInnhold.tsx

```diff
@@ -0,0 +1,326 @@
+            {arbeidsgiverOpplysningerPerId[inntektsmelding.arbeidsgiverIdent]?.navn ?? '-'}
+          </span>
+          <span>
+            <FormattedMessage id="InntektsmeldingFaktaPanel.arbeidsgiver.underenhet" />:{' '}
+            {inntektsmelding.arbeidsgiverIdent}{' '}
+          </span>
+        </InntektsmeldingInfoBlokk>
+
+        <InntektsmeldingInfoBlokk
+          tittel={intl.formatMessage({ id: 'InntektsmeldingFaktaPanel.kontaktperson.heading' })}
+        >
+          <span>
+            <FormattedMessage id="InntektsmeldingFaktaPanel.kontaktperson.navn" />: {inntektsmelding.kontaktpersonNavn}
+          </span>
+          <HStack align="start" gap="2">
+            <span>
+              <FormattedMessage id="InntektsmeldingFaktaPanel.kontaktperson.telefon" />{' '}
+              {inntektsmelding.kontaktpersonNummer}
+            </span>{' '}
+            <CopyButton size="xsmall" copyText={inntektsmelding.kontaktpersonNummer} />{' '}
+          </HStack>
+        </InntektsmeldingInfoBlokk>
+
+        <InntektsmeldingInfoBlokk
+          tittel={intl.formatMessage({ id: 'InntektsmeldingFaktaPanel.månedsinntekt.heading' })}
+        >
+          <span>{formatCurrencyWithKr(inntektsmelding.inntektPrMnd)}</span>
+          {/*TODO: Få inn endringsgrunn*/}
+        </InntektsmeldingInfoBlokk>
+
+        <InntektsmeldingInfoBlokk
+          tittel={intl.formatMessage(
+            { id: 'InntektsmeldingFaktaPanel.startDato.heading' },
+            {
+              ytelse: getKodeverknavnFraKode(
+                alleKodeverk,
+                KodeverkType.FAGSAK_YTELSE,
+                fagsak.fagsakYtelseType,
+              ).toLowerCase(),
+            },
+          )}
+        >
+          <span>
+            {inntektsmelding.startDatoPermisjon ? <DateLabel dateString={inntektsmelding.startDatoPermisjon} /> : '-'}
+          </span>
+          <span>
+            <FormattedMessage id="InntektsmeldingFaktaPanel.startDato.fraArbeidsgiver" />
+          </span>
+        </InntektsmeldingInfoBlokk>
+
+        <InntektsmeldingInfoBlokk
+          tittel={intl.formatMessage({ id: 'InntektsmeldingFaktaPanel.kilde.heading' })}
+        >
+          <KildeSystem inntektsmelding={inntektsmelding} />
+        </InntektsmeldingInfoBlokk>
+
+        <InntektsmeldingInfoBlokk
+          tittel={intl.formatMessage({ id: 'InntektsmeldingFaktaPanel.refusjon.heading' })}
+        >
+          <Refusjon inntektsmelding={inntektsmelding} />
+        </InntektsmeldingInfoBlokk>
+        <BortfalteNaturalYtelser inntektsmelding={inntektsmelding} />
+        <BehandlingsOversikt
+          alleKodeverk={alleKodeverk}
+          inntektsmelding={inntektsmelding}
+          alleBehandlinger={alleBehandlinger}
+          behandling={behandling}
+        />
+      </HGrid>
+    </VStack>
+  );
+};
+
+const KildeSystem = ({ inntektsmelding }: { inntektsmelding: Inntektsmelding }) => {
+  if (inntektsmelding.kildeSystem.toUpperCase() === 'NAV_NO') {
```

Comment:
om kildeSystem er eit kodeverk så brukar me vanlegvis å legga definisjonen for dette i kodeverk-pakka. Finn dog berre desse konstantane i JournalKanal-pakka. Veit ikkje om dette er det same kodeverket. Høyr evt med PEtter

Reply:
Om det ikkje er eit kodeverk så passar det ikkje inn i kodeverk-pakka, så da er denne måten grei